openlibrary/plugins/upstream/account.py

```diff
@@ -55,6 +55,7 @@
         "username_registered": "This username is already registered",
         "ia_login_only": "Sorry, you must use your Internet Archive email and password to log in",
         "max_retries_exceeded": "A problem occurred and we were unable to log you in.",
+        "invalid_s3keys": "Login attempted with invalid IA s3 credentials.",
```

Comment:
If this is a public error message, spell out "IA" ==> "Internet Archive"

Reply:
Will change